bin/generate_contributors.go

```diff
@@ -120,6 +90,37 @@ func main() {
 			}
 			page++
 		}
+
+		// Get all created PR contributors
+		prRangeOk = true
+		page = 0
+
+		for page < 15 && prRangeOk{
```

Comment:
Why 15? Just an upper bound?

Reply:
Yeah, I kept this from the original script - is there a better way to manage this?

Reply:
Is this just to stop a runaway loop? Why not just let the API dictate when the results are done (you already break when the issue/pr is too old)?

Reply:
Yeah, I think that's why it was there before. Good point, will fix!